web-ui/representation/dune

```diff
@@ -0,0 +1,6 @@
+(library
+ (public_name ocaml-ci-web.representation)
```

Comment:
Not crazy on this naming what about `model` or `types` ?
Not really better names but they are shorter 🤣